product_to_bizerba_scale/views/product_template_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page name="scale" string="Scale">
+                    <group name="scale_communication" string="Scale Communication">
+                        <group>
+                            <!-- <field name="scale_group_id" attrs="{'required':[('to_weight','=',True)]}"/> -->
```

Comment:
why commenting or it was already commented ?

Reply:
it was already commented in [product_to_scale_bizerba_extended (9.0)](https://github.com/coopiteasy/addons/blame/9.0/product_to_scale_bizerba_extended/views/product_view.xml#L12)

Reply:
Remove it. ;)

Reply:
done